packages/twenty-server/src/database/typeorm/metadata/migrations/1723038077987-addSoftDelete.ts

```diff
@@ -0,0 +1,17 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddSoftDelete1723038077987 implements MigrationInterface {
+  name = 'AddSoftDelete1723038077987';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "metadata"."objectMetadata" ADD "softDelete" boolean`,
```

Comment:
Should we use a verb to remain consistent with other booleans? e.g. `isSoftDeleteEnabled` or `isSoftDeletable`)